controllers/recipesController.js

```diff
@@ -2,23 +2,26 @@ const Recipe = require('../models').Recipe
 const Key = require('../models').Key
 const pry = require('pryjs');
 const fetch = require('node-fetch');
+var url = require('url');
 
 const create = async (req, res) => {
+  var url_parts = url.parse(req.url, true);
```

Comment:
Did you try using `req.params.ingredient`?

Reply:
I totes tried but I was getting a blank object for req.params. Shrug.